ghost-cat-app/src/app/View/image-thumbnails-page/image-thumbnails-page.component.html

```diff
@@ -1 +1,39 @@
-<p>image-thumbnails-page goes here</p>
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" href="image-thumbnails-page.component.css">
+</head>
+<body>
+        <div id="block_container">
+                <div id ="previousPage" >
+                    <button class="button" type="button">Previous Page</button>
+                </div>
+               <div id ="nextPage" >
+                   <button class="button" type="button">Next Page</button>
+               </div>
+           </div>
+           
+    <div>
+        <div id ="currentQuery" >
+            <button class="button" type="button">Current Filters</button>
+        </div>
+        <div id ="changeQuery" >
+            <button class="button" type="button">Change Filters</button>
+        </div>
+    </div>
+
+    <p><i class="arrow right"></i></p>
+    <p><i class="arrow left"></i></p>
+    <div class ="wrapper" >
+        <div  *ngFor="let image of images" >
+            <img src='{{image}}'/>
+        </div>
+    </div>
+
```

Comment:
If you update this file to get rid of some of those tags, it might be worth getting rid of some of this extra whitespace too.